FoodFriends/app/src/main/java/com/example/foodfriends/misc/RestaurantServer.java

```diff
@@ -177,16 +184,22 @@ public void done(List<Restaurant> restaurants, ParseException e) {
                         parseSource = false;
                     }
                     if (restaurants.size() == 0) {
-                        yelpQuery(apiKey);
+                        yelpQuery(apiKey, listener);
```

Comment:
are we storing apiKey as plain text? Not asking you to encrypt it but wondering if u have thought about how a real world production system would do it

Reply:
storing apiKey in a secrets file as plain text! Not sure how it is done in production, but will look into it.

Reply:
This makes sense: 
"Instead of embedding your API keys in your applications, store them in environment variables or in files outside of your application's source tree. Do not store API keys in files inside your application's source tree."